vendor/vendor-ios/src/main/kotlin/com/malinskiy/marathon/ios/AppleVendor.kt

```diff
@@ -34,6 +34,8 @@ val AppleVendor = module {
     }
     single<TestParser?> { AppleTestParser(get(), get(), get()) }
     single<MarathonLogConfigurator> { AppleLogConfigurator(get()) }
-    single<TestBundleIdentifier?> { AppleTestBundleIdentifier() }
-    single<AppleTestBundleIdentifier> { AppleTestBundleIdentifier() }
```

Comment:
This mistakenly created 2 bundle identifiers and hence produced "no test bundle identifier" errors in runtime